src/tests/ftest/launch.py

```diff
@@ -521,7 +521,7 @@ def get_test_list(tags):
         command = ["avocado", "list", "--paginator=off"]
         for test_tag in test_tags:
             command.append(str(test_tag))
-        command.extend(test_list)
+        command.extend(test_list if test_list else ["./"])
```

Comment:
Isn't `test_list` guaranteed to be at least `./` here by lines 519-520?

Just concerned about some future change only changing one of lines 520 or 524 and not both.